tests/components/ecobee/test_number.py

```diff
@@ -0,0 +1,117 @@
+"""The test for the ecobee thermostat number module."""
+from datetime import timedelta
+from unittest.mock import patch
+
+from pytest import MonkeyPatch
+
+from homeassistant.components.ecobee.const import DOMAIN as ECOBEE_DOMAIN
+from homeassistant.components.number import ATTR_VALUE, DOMAIN, SERVICE_SET_VALUE
+from homeassistant.const import ATTR_ENTITY_ID, UnitOfTime
+from homeassistant.core import HomeAssistant
+from homeassistant.util import dt
+
+from .common import setup_platform
+
+from tests.common import async_fire_time_changed
+
+VENTILATOR_MIN_HOME_ID = "number.ecobee_ventilator_min_time_home"
+VENTILATOR_MIN_AWAY_ID = "number.ecobee_ventilator_min_time_away"
+THERMOSTAT_ID = 0
+
+
+async def test_ventilator_min_on_home_attributes(hass):
+    """Test the ventilator number on home attributes are correct."""
+    await setup_platform(hass, DOMAIN)
+
+    state = hass.states.get(VENTILATOR_MIN_HOME_ID)
+    assert state.state == "20"
+    assert state.attributes.get("min") == 0
+    assert state.attributes.get("max") == 60
+    assert state.attributes.get("step") == 5
+    assert state.attributes.get("friendly_name") == "ecobee Ventilator min time home"
+    assert state.attributes.get("unit_of_measurement") == UnitOfTime.MINUTES
+
+
+async def test_ventilator_min_on_away_attributes(hass):
+    """Test the ventilator number on away attributes are correct."""
+    await setup_platform(hass, DOMAIN)
+
+    state = hass.states.get(VENTILATOR_MIN_AWAY_ID)
+    assert state.state == "10"
+    assert state.attributes.get("min") == 0
+    assert state.attributes.get("max") == 60
+    assert state.attributes.get("step") == 5
+    assert state.attributes.get("friendly_name") == "ecobee Ventilator min time away"
+    assert state.attributes.get("unit_of_measurement") == UnitOfTime.MINUTES
+
+
+async def test_set_min_time_home(hass: HomeAssistant, monkeypatch: MonkeyPatch):
+    """Test the number can set min time home."""
+    target_value = 40
+    with patch(
+        "pyecobee.Ecobee.set_ventilator_min_on_time_home"
```

Comment:
```suggestion
        "homeassistant.components.ecobee.Ecobee.set_ventilator_min_on_time_home"
```

We should patch the object where its imported